### PR TITLE
Add Discord icon to the landing page navbar

### LINKED ISSUE #1037: Add Discord icon to the top of the website
Right now, the menu on the top of the website only includes a link to the fairlearn GitHub repo. 
image

Discord is where the fairlearn community mostly communicates, so we should add the [Discord icon](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.svgrepo.com%2Fsvg%2F353655%2Fdiscord-icon&psig=AOvVaw1i86aATKswyKDovRBr90pM&ust=1646405362654000&source=images&cd=vfe&ved=2ahUKEwj99dKHmKr2AhVoGVkFHVSUBnQQr4kDegUIARDdAQ) to the top of the page so new contributors can quickly get linked up with the rest of the team! This issue addresses a comment raised by @romanlutz  in PR #1036.

### PR DESCRIPTION
**Pull request by iofall**
Fixes #1037 

**Changes introduced in this PR:**
- Change `.navbar li` font-size to `16px` at the original `18px` size, the navbar looked congested after adding the button.
- Add `Button_Discord_Default.svg` and `Button_Discord_Hover.svg` files.
- Modify `index.html` to show the `Discord Button` in the navbar.
- Remove `box-shadow` for the buttons.

### CODE REVIEW

File: docs/static_landing_page/css/landing_page_style.css
Comment (romanlutz) [inline docs/static_landing_page/css/landing_page_style.css:152]: What does it look like if you don't make this change? I'm trying to understand the reasoning behind this. Thanks!!!

Comment (iofall) [reply docs/static_landing_page/css/landing_page_style.css:152]: This is at `18px`:

image

`16px`: [Link](https://4917-133444044-gh.circle-artifacts.com/0/docs/_build/html/index.html)

image

Comment (iofall) [reply docs/static_landing_page/css/landing_page_style.css:152]: The difference does not come through well in the images. You could use the inspector to change the font size and view in the browser itself using the Circle CI link.

Comment (romanlutz) [reply docs/static_landing_page/css/landing_page_style.css:152]: Is the purpose of the change to create a gap between the text items and the buttons?

Comment (iofall) [reply docs/static_landing_page/css/landing_page_style.css:152]: Yes 👍

Comment (romanlutz) [reply docs/static_landing_page/css/landing_page_style.css:152]: I suppose I agree with @MiroDudik in that we'd need someone with UI expertise to tell us if that's a good solution. @nessamilan perhaps?

Comment (nessamilan) [reply docs/static_landing_page/css/landing_page_style.css:152]: Thanks for the tag! 

**A few thoughts:**
- In practice, font-size changes should be made as a global change not for one instance, so it maintains a consistent type-size system throughout the site.
- I wouldn't recommend 2 buttons in the top nav. For UI, it's recommended to have a max 5-7 text links in a top navigation. Buttons are reserved for Login or one 3rd party partner (in Fairlearn's case Github as the primary creation/community platform). For example, where both Github and Login exist, it's common to see "Github" as a text link and "Login" as a button.

**Recommendation:**
- Update all 18px fonts to 16px (@iofall which looks really good in your screenshot, btw!).
- Use and icon for each Github and Discord with no label or button background. 
_Note: this subsequently makes room for a 3rd icon in the future. I'd cap these at three, more communities/social entities can be featured in the footer._
- Add tooltips for each icon.
- The Discord color looks really cool. If you decide on using the color, I'd update Github as well so they are both their brand colors. Based on the inspect tool, it looks like Github's header color is #24292f.

Comment (romanlutz) [reply docs/static_landing_page/css/landing_page_style.css:152]: Thanks @nessamilan !
The only thing I'd add is that we already use the colors in the generated sphinx header, but only on hover. By default the icons are gray or black I think. You can find the colors there to make sure they match, @iofall 🙂

Comment (iofall) [reply docs/static_landing_page/css/landing_page_style.css:152]: Thank you for the feedback @nessamilan. I think for this issue, I will modify both the logos (GitHub and Discord) and make them as you have suggested.

For the font size, I see that there are other design issues you have opened. I could help with those. Modifying the font size would be better tackled in those. 

> * The Discord color looks really cool. If you decide on using the color, I'd update Github as well so they are both their brand colors.

Yes, it is the official color for Discord. For GitHub, it would just be plain black and white.

File: docs/static_landing_page/index.html
Comment (romanlutz) [inline docs/static_landing_page/index.html:28]: I like your screenshot of what it looks like. I'm just not sure of what all these options do. Could you explain what mx or px (and lower down py-2) mean? If you are well-versed in this, perhaps add a comment into the HTML? I think this might end up being very useful long-term as we have to maintain this. If you don't know that's also fine. I'll have to run it locally to confirm in that case, but I can do that.

Comment (iofall) [reply docs/static_landing_page/index.html:28]: Actually `mx`, `px` and `py` etc are built-in Bootstrap classes. We don't need comments for these, they are standard classes that are provided by bootstrap. We are already using many of these classes across the whole page and anyone who needs to read into them could look at the Bootstrap documentation or the generated CSS.

Here's a short explanation:
1. `mx` stands for `margin-x` so it will add a margin in the x-axis both on the left and the right. Similarly, you can use `mr` to specify `margin-right` and `ml` for `margin-left`. By using `mx` instead of `mr` in the line 28, we are able to center the whole nav content.
2. `px` also works in a similar fashion but for padding. `py` will specify the padding on the y-axis that is - `padding top` and `padding bottom`. I added this `py` so that when the navbar collapses, there is some vertical gap between the two icons for GitHub and Discord.

File: docs/static_landing_page/css/landing_page_style.css
Comment (iofall) [reply docs/static_landing_page/css/landing_page_style.css:398]: If you zoom into our home page right now, near the GitHub button in the navbar you will see a box with a shadow.

**Image for reference**
image

This box is created by the line 398. We don't really need it. 

Additionally, in the collapsed / mobile version it creates the same effect but more pronounced as seen here:

image

In our PR since we are using multiple logos, the box shadow creates a rectangular shadow around them which although subtle creates a striped effect in the collapsed navbar.

File: docs/static_landing_page/index.html
Comment (romanlutz) [reply docs/static_landing_page/index.html:28]: You'd be amazed how much people without front end experience struggle to understand this code 😄 [I am very much looking at myself with this comment.]

Even just a comment to the effect of "This code uses bootstrap so check out the documentation for details <link>" goes a long way.

Thanks for explaining!

**Review by romanlutz — APPROVED** [review]

File: docs/static_landing_page/css/landing_page_style.css
Comment (nessamilan) [reply docs/static_landing_page/css/landing_page_style.css]: This looks excellent, @iofall !! Thanks for doing this work.  Sounds good about the font sizes in another issue. I'll go back over the list of design issues and see where it fits. You're help would be greatly appreciated!

**Comment by romanlutz** [comment]
Any other feedback from @fairlearn/fairlearn-maintainers ? We need a second approval (if there are no objections) to move forward.

**Pull request merged**
